### PR TITLE
Updating control to check ports and services for all zones.

### PR DESCRIPTION
**Pull request by djhaynes**
@trevor-vaughan I took a pass at this and wrote the check so it checks all zones rather than just the default. This is part of PR #8.

### CODE REVIEW

File: controls/V-72219.rb [hunk 1]
```diff
@@ -1,7 +1,65 @@
...
+    # Examples (zone:[ports])
+    #"public"=>['12345/tcp', '23456/tcp'],
+    # "dmz"=>['80/http', '22/ssh']
+  },
+  description: "Ports that firewalld should be configured to deny."
+)
+
+default_firewalld_zone = input(
+  'default_firewalld_zone',
+  value: "public",
```
Comment (trevor-vaughan) [inline controls/V-72219.rb]: Why does this need to be your default zone?

Comment (djhaynes) [reply controls/V-72219.rb]: What would you prefer for the default zone?

Comment (trevor-vaughan) [reply controls/V-72219.rb]: @djhaynes It should just be discovered shouldn't it? The name is irrelevant as long as the settings are correct so I don't think that it's something that needs to be checked at all.

Comment (djhaynes) [reply controls/V-72219.rb]: I removed the input and the check for the default firewall zone.

File: controls/V-72219.rb [hunk 2]
```diff
@@ -1,7 +1,59 @@
...
+    # 'external',
+    # 'dmz',
+    # 'work',
+    # 'home',
+    # 'internal',
+    # 'trusted'
+  ]
+)
+
+iptable_rules = input(
```
Comment (trevor-vaughan) [inline controls/V-72219.rb]: Should this be `iptables` instead of `iptable`?

Comment (djhaynes) [reply controls/V-72219.rb]: Just fixed the typo.

Comment (djhaynes) [reply controls/V-72219.rb]: @trevor-vaughan  Can we merge this control in now?

**Pull request merged**